css/components.css

```
@@ -2030,4 +2197,318 @@
+}
+
+.event-section:last-of-type {
+  border-bottom: none;
+}
+
+/* Event Headers */
+.event-header {
+  text-align: center;
+  margin-bottom: var(--space-3xl);
+  max-width: 800px;
+  margin: 0 auto var(--space-3xl);
```

Biome (2.1.2)

[error] Unexpected shorthand property margin after margin-bottom



(lint/suspicious/noShorthandPropertyOverrides)